Update socialmedia.md (#44307)
fleetdm/fleet: handbook/marketing/socialmedia.md

## Strategy

LinkedIn is Fleet's primary paid social channel and is treated as a **brand awareness and engagement play**, not a direct-attribution pipeline channel.

- **Primary metric:** Click-through rate (CTR). CTR is the CEO's proxy for creative resonance.  
- **Vanity metrics:** Impressions and engagement. No one is held accountable for LinkedIn ROI.  
- **Accountability:** The CEO holds people accountable for failing to execute things he considers logically sound or has seen work before, and for failing to act ASAP on post suggestions he drops in \#help-marketing.   
- **Creative freshness:** Ads must reflect Fleet's current positioning and messaging. Exception: an ad older than 6 months may stay if it still has strong CTR/engagement, and still reflects Fleet’s messaging.  
- **Prioritization:** Fix known issues before experimenting with new channels or tactics, e.g., new campaigns being created, different audiences being used

### Target personas

|  | Santas (decision-makers) | Elves (practitioners) |
| :---- | :---- | :---- |

> | Titles | IT Directors, VPs of IT, CIOs, CISOs | Sysadmins, platform engineers, compliance analysts |

Reviewer: - Remove CISOs, CIOs, add Head of I&O, add Head of Digital Workplace
- Change "Titles" to "Example titles"
- Replace Platform Engineers with Client Platform Engineers, remove Compliance Analysts

| Care about | ROI, consolidation, risk reduction, compliance | Workflow, automation, cross-platform, open source |
| LinkedIn audience | santa.it.major.mdm.strict | elf.it.major.mdm.strict |

### Messaging pillars

> All creatives must map to one or more: high-agency device management; infrastructure as code / GitOps; AI-accelerated IT (humans in the loop); cross-platform and open source (macOS, Windows, Linux, ChromeOS — Android is not yet promoted); competitive displacement vs. Jamf, Intune, and Workspace ONE.

Reviewer: All creatives must map to one of the 2 active campaign themes as indicated in the horizons tab of the OKRs document (we can link to that confidential google doc instead of duplicating this info here)

## Responsibilities

> | Role | Owns |

Reviewer: ```suggestion
| Role | DRI of... |
```

| :---- | :---- |
| Content Specialist (Marketing DRI) | Campaign Manager day-to-day, daily health checks, promote/pause/retest decisions, budget pacing, \#help-marketing monitoring, Social Media Manager coordination. |
| Social Media Manager | Daily posting, boosting per calendar or direction, content calendar updates, tagging Fleeties for engagement in \#help-marketing, LinkedIn comment response. |
| CMO | Ad strategy and creative direction, monthly performance review, final call on budget allocation, and major strategy changes. |

> | CEO | Early monthly reviews (transitioning out by the end of Q2 2026), directing specific experiments or spend pushes, and owning saved audience definitions. |

Reviewer: ```suggestion
| CEO | Saved audience definitions |
```

| Sales | Triggers ABM campaigns via Salesforce. |
| Ad agency | Executes daily/weekly operations per this page, produces creative, provides weekly reports, and flags anomalies to CMO. |

> | Head of Design | Approves images/designs before they go live. |

Reviewer: ```suggestion
| Head of Design | Approves new images/designs on creatives posted by the company page before they receive significant spending |
```

## Campaign structure

Fleet runs three campaign types in LinkedIn Campaign Manager. 

### Evergreen (engagement)

Always-on. Only proven creatives. Engagement objective.

> - **Entry requirement:** Every creative must first earn 2%+ CTR in Targeting Experiments.  

Reviewer: ```suggestion
- **Entry requirement:** Every creative must first earn 1%+ CTR (4%+ for non-company page posts) in "Targeting experiments".  
```

- **Refresh:** quarterly — remove stale, add new proven, align to current messaging.  
- **Budget split:** unaware.elf.it-major-mdm.strict 60%,  unaware.santa.it-major-mdm.strict, unaware.elf.it-gapfiller 20%.

> ### Targeting Experiments

Reviewer: ```suggestion
### Targeting experiments
```

Where everything is tested first.

- **Run length:** 2–3 days.  
- **Default audience:** elf.it.major.mdm.strict. Use santa.it.major.mdm.strict for executive-focused content.  
- **Budget flex:** Borrow from Evergreen if needed. The CEO may direct pushes for high-conviction posts.

**CTR decision thresholds:**

| CTR | Verdict | Action |
| :---- | :---- | :---- |
| < 0.65% | Not performing | Pause. Do not reuse. Never delete — keep the data. |
| 0.65%–2% | Gray zone | Decide with CMO: retest, move to Evergreen, or drop. |
| 2%+ | Strong | Promote to Evergreen (or extend run spend). |
| 10%+ | Exceptional | Keep running as-is in Evergreen until performance stabilizes or the messaging is no longer relevant. |

Context: ~0.4% is typical for company page posts; 1–4%+ is strong for personal posts.

> ### Automated campaigns (ABM from Salesforce)

Reviewer: Comment this section out for now

2. Select audience: Default to elf.it.major.mdm.strict; use santa.it.major.mdm.strict only for executive-focused content.  
3. Set duration: 2–3 days, starting Tuesday or Thursday. Events, happy hours, and workshops may run longer at the CMO's direction.  
4. Set budget: 2–3% of the monthly budget over the run, or follow CEO's direction for high-conviction.  
5. Launch inside the **Targeting Experiments** campaign. Never boost directly from the company page \- always route through Campaign Manager so performance is tracked.  
6. After the run, apply the CTR decision thresholds. Log the result. Promote to Evergreen or pause, never delete.

### How to promote a creative to Evergreen

1. Confirm the creative hit 2%+ CTR (or 10%+ for exceptional cases) in Targeting Experiments.  
2. Pick the right segment: practitioner content → unaware.elf.it-major-mdm.strict; executive → unaware.santa.it-major-mdm.strict; broad → unaware.elf.it-gapfiller.  
3. Add the creative to that segment. Leave the original paused in Targeting Experiments as a record.  
4. Monitor over the first week (2–3 check-ins) to confirm it holds up at scale.

> ## Creative types

Reviewer: Have we tested these concepts like quote cards before? Or is this a brainstorm of new things we could try?

Author: @mikermcneil this is something I am currently trying so I have a process to follow and can keep track of what's happening for my eyes :)

- **Thought leader ads**: Text-heavy, face-based leadership ads. Must reflect current positioning; refresh text-heavy ads that don't stand out with visual elements.  
- **Event and workshop promotions**: GitOps workshops, happy hours, conferences. Post-event summaries come from attending Fleeties and from the Social Media Manager (organic post from Fleet's page, then boosted). May run longer than the standard 2–3 day window.

## Guardrails

| Always | Never |
| :---- | :---- |
| Test everything in Targeting Experiments before Evergreen | Put untested creative directly into Evergreen |
| Check ABM daily for empty creative slots | Let ABM campaigns run without creatives |
| Pause creatives below 0.65% CTR quickly | Delete past experiments, campaigns, or data |
| Keep Evergreen aligned to current Fleet messaging | Change saved audiences without CEO design review |
| Boost through Campaign Manager for tracking | Boost directly from the LinkedIn company page |
| Default to elf.it.major.mdm.strict when unsure | Run experiments over weekends |
| Log every experiment result, even failures | Overspend without pulling from Evergreen first |

<meta name="maintainedBy" value="akuthiala">

> <meta name="title" value="🫧 Fleet social media">

Reviewer: Why remove the title?

Author: It kept having an error/fail code when I tried to make the pull request and this is what Claude told me to do lol. Should I put <meta name="title" value="🫧 Fleet social media"> this back in as the meta tag?

Reviewer: Instead of deleting the title, we can review the error message and make it work.  Try checking other similar pages

Author: Thank you! I'll have a gander!
